README.md

```diff
@@ -620,6 +620,107 @@ protected const LOAD = [
 php app.php queue:pause local
 ```
 
+### TCP
+
+RoadRunner includes TCP server and can be used to replace classic TCP setup with much greater performance and flexibility.
+
+#### Bootloader
+
+Add `Spiral\RoadRunnerBridge\Bootloader\TcpBootloader` to application bootloaders list:
+```php
+use Spiral\RoadRunnerBridge\Bootloader as RoadRunnerBridge;
+
+protected const LOAD = [
+    // ...
+    RoadRunnerBridge\TcpBootloader::class, 
+    // ...
+];
+```
+
+This bootloader adds a dispatcher and necessary services for TCP to work.
+Also, using the `addService` and `addInterceptors` methods can dynamically add services to TCP servers and configure interceptors.
+
+#### Configuration
+
+Configure `tcp` section in the RoadRunner `.rr.yaml` configuration file with needed TCP servers. Example:
+
+```yaml
+tcp:
+    servers:
+        tcp_access_point_1:
+            addr: tcp://127.0.0.1:7777
+            delimiter: "\r\n" # by default
+        server2:
+            addr: tcp://127.0.0.1:8889
+
+    pool:
+        num_workers: 2
+        max_jobs: 0
+        allocate_timeout: 60s
+        destroy_timeout: 60s
+```
+
+Create configuration file `app/config/tcp.php`. In the configuration, it's required to specify the services that 
+will handle requests from a specific TCP server. Optionally, interceptors can be added for each specific server. 
+With the help there, can add some logic before handling the request in service. Configuration example:
+
+```php
+<?php
+
+declare(strict_types=1);
+
+return [
+    /**
+     * Services for each server.
+     */
+    'services' => [
+        'tcp_access_point_1' => SomeService::class,
+        'server2' => OtherService::class
+    ],
+
+    /**
+     * Interceptors, this section is optional.
+     */
```

Comment:
Add link to interceptors 
https://spiral.dev/docs/cookbook-domain-core/2.8/en#core-interceptors

Reply:
Added